subgraphs/apeswap/src/mappings/pool.ts

```diff
@@ -0,0 +1,340 @@
+    let amountInNativeToken = token0._derivedNativeToken.times(amount0Total);
+    swap.amountInUSD = amountInNativeToken.times(helperStore._value);
+    swap.amountOut = toBigInt(amount1Total, token0.decimals);
+    let amountOutInNativeToken = token1._derivedNativeToken.times(amount1Total);
+    swap.amountOutUSD = amountOutInNativeToken.times(helperStore._value);
+    swap.save();
+
+    // Take a PoolDailySnapshot
+    createPoolDailySnapshot(event.address, event.block.number, event.block.timestamp, pool);
+
+    // Take FinancialsDailySnapshot
+    let financials = getOrCreateFinancials(protocol, event.block.timestamp, event.block.number);
+    let tradingFee = getOrCreateTradingFees(event.address);
+    let protocolFee = getOrCreateProtocolFeeShare(event.address);
+    let supplyFee = getOrCreateSupplyFeeShare(event.address);
+
+    let tradingFeeAmount = amount0Total.times(tradingFee.feePercentage);
+
+    let tradingFeeAmountUSD: BigDecimal = tradingFeeAmount.times(token0._derivedNativeToken).times(helperStore._value);
+    let protocolFeeAmountUSD: BigDecimal = tradingFeeAmountUSD.times(protocolFee.feePercentage);
+    let supplyFeeAmountUSD = tradingFeeAmountUSD.times(supplyFee.feePercentage);
+
+    financials.totalValueLockedUSD = pool.totalValueLockedUSD;
+    financials.totalVolumeUSD = pool.totalVolumeUSD;
+    financials.feesUSD = financials.feesUSD.plus(tradingFeeAmountUSD);
+    financials.supplySideRevenueUSD = financials.supplySideRevenueUSD.plus(supplyFeeAmountUSD);
+    financials.protocolSideRevenueUSD = financials.protocolSideRevenueUSD.plus(protocolFeeAmountUSD);
+    financials.save();
+
+    // Take UsageMetricsDailySnapshot
+    updateUsageMetrics(event.params.sender, protocol, event.block.timestamp, event.block.number);
+  }
+}
+
+export function handleSync(event: Sync): void {
+  log.info("Sync mapping on pool {}", [event.address.toHexString()]);
+  let reserve0 = event.params.reserve0;
+  let reserve1 = event.params.reserve1;
+  let id = event.address.toHexString();
+
+  let protocol = getOrCreateProtocol();
+  let pool = LiquidityPool.load(id);
+  if (pool !== null && reserve0 !== BIGINT_ZERO && reserve1 !== BIGINT_ZERO) {
+    // reset factory liquidity by subtracting only tracked liquidity
+    protocol._totalValueLockedInNativeToken = protocol._totalValueLockedInNativeToken.minus(
+      pool._trackedNativeTokenReserve,
+    );
+
+    // reset token total liquidity amounts
+    let token0 = Token.load(pool._token0)!;
+    let token1 = Token.load(pool._token1)!;
+
+    pool._reserve0 = toDecimal(reserve0, token0.decimals);
+    pool._reserve1 = toDecimal(reserve1, token1.decimals);
+
+    if (pool._reserve1.notEqual(BIGDECIMAL_ZERO)) pool._token0Price = pool._reserve0.div(pool._reserve1);
+    else pool._token0Price = BIGDECIMAL_ZERO;
+    if (pool._reserve0.notEqual(BIGDECIMAL_ZERO)) pool._token1Price = pool._reserve1.div(pool._reserve0);
+    else pool._token1Price = BIGDECIMAL_ZERO;
```

Comment:
proper formatting

Reply:
The printWidth here refers to the prettier printWidth

Reply:
same here,
```
if () {
} else {
}

Reply:
Fixed